### DISCUSSION TITLE
Can't open my Data set in Shiny app dashboard

### DISCUSSION THREAD

**Discussion by RamadhanRC** [teamalgoritma/community#413] [Q&A] [answered]
Dear all,

Saya sudah memasukan dataset saya  pada bagian server.r dan ui.r tetapi saat di run app data set tersebut tidak muncul pada tab menu manapun mohon bantuannya

Pada UI.R
`
tabItem(tabName = "menu_3",
            DT::dataTableOutput(outputId = "table"))
`

Pada Server.R
`
output$table <- renderDataTable({
    datatable(compsal,
              options = list(scrollX = TRUE))})
`

**Comment by kinantytasyao** [comment, marked as answer]
Dear Bapak @RamadhanRC,
 
Bapak seharusnya memasukkan dataTableOutput ke dalam parameter fluidRow. fluidRow berfungsi untuk mengatur layout berdasarkan baris. Lebih lengkapnya Bapak dapat membaca referensi berikut terkait Layout di shiny: http://rstudio.github.io/shinydashboard/structure.html

Sehingga kode Bapak menjadi:
`
tabItem(tabName = "menu_3",
                    fluidRow(
                        box(width = 12,
                            dataTableOutput(outputId = "table"))
                    ))
`

Demikian yang dapat kami sampaikan, semoga membantu.

**Reply by RamadhanRC** [reply]
Tapi dari yang di contohkan saat kelas IP ko ga pakai fluidrownya ya?

**Reply by RamadhanRC** [reply]
Sudah saya coba menambahkan fluidrow tetap tidak mau tampil ya

**Reply by kinantytasyao** [reply]
karena kode Bapak sebenarnya sudah benar dan di kelas tidak ada masalah ketika deploy ke shiny, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
